xbuild/src/cargo/mod.rs

```diff
@@ -465,7 +465,10 @@ impl CargoBuild {
     }
 
     pub fn exec(mut self) -> Result<()> {
-        self.cargo_target_env("RUSTFLAGS", &self.rust_flags.clone());
+        // TODO: There's no target-triple specific ENCODED_RUSTFLAGS: https://github.com/rust-lang/cargo/issues/14398
```

Comment:
It's not clear to me why we need a target triple specific version of this flag, can't we use the regular version?